external/storm-kafka-monitor/src/main/java/org/apache/storm/kafka/monitor/KafkaOffsetLagUtil.java

```diff
@@ -89,6 +89,10 @@ public static void main (String args[]) {
                     printUsageAndExit(options, OPTION_ZK_SERVERS_LONG + " and " + OPTION_ZK_COMMITTED_NODE_LONG + " are required  with " +
                             OPTION_OLD_CONSUMER_LONG);
                 }
+                String zkNode = commandLine.getOptionValue(OPTION_ZK_COMMITTED_NODE_LONG);
```

Comment:
I am not sure if just checking the length <= 1 is really that useful. As you mentioned that it should start with / may be we add check satisfying all the conditions for a valid zookeeper node name? Or just remove any check? We already have a check to see if node exists later.